src/main/java/de/mediathekview/mserver/crawler/wdr/tasks/WdrTopicOverviewTask.java

```diff
@@ -0,0 +1,56 @@
+package de.mediathekview.mserver.crawler.wdr.tasks;
+
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractDocumentTask;
+import de.mediathekview.mserver.crawler.basic.AbstractUrlTask;
+import de.mediathekview.mserver.crawler.basic.TopicUrlDTO;
+import de.mediathekview.mserver.crawler.wdr.WdrTopicUrlDto;
+import de.mediathekview.mserver.crawler.wdr.parser.WdrTopicOverviewDeserializer;
+import java.util.List;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import org.jsoup.nodes.Document;
+
+public class WdrTopicOverviewTask extends AbstractDocumentTask<TopicUrlDTO, WdrTopicUrlDto> {
+
+  private final int recursiveCount;
+
+  public WdrTopicOverviewTask(
+      AbstractCrawler aCrawler,
+      ConcurrentLinkedQueue<WdrTopicUrlDto> aUrlToCrawlDtos,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDtos" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)